wgsl/index.bs

```diff
@@ -2082,6 +2082,18 @@ shift_expression
   | shift_expression SHIFT_RIGHT additive_expression
         OpShiftRightLogical or OpShiftRightArithmetic
 
+and_expression
+  : equality_expression
```

Comment:
This should point at the `shift_expression`

Reply:
@dj2 Fixed.